src/bootstrap/dist.rs

```diff
@@ -227,6 +227,8 @@ fn make_win_dist(
         "libwsock32.a",
         "libdbghelp.a",
         "libmsimg32.a",
+        "libcredui.a",
+        "libsecur32.a",
```

Comment:
This list used to be in alphabetical order. I sure do wish it could continue to be in alphabetical order.